diary-volumes/volume3.xml

```diff
@@ -7,7 +7,7 @@
           <persName ref="#Andrews_Emma_B">Emma B. Andrews</persName>
```

Comment:
@emmafritzberg scrap vol 16 changes in this MR, but keep vol 3 (needs a once-over for correctness)